react/index.d.ts

```diff
@@ -2140,60 +2140,237 @@ declare namespace React {
         unselectable?: boolean;
     }
 
+    // this list is "complete" in that it contains every SVG attribute 
+    // that React supports, but the types can be improved.
+    // Full list here: https://facebook.github.io/react/docs/dom-elements.html
+    //
+    // The three broad type categories are (in order of restrictiveness):
+    //   - "number | string"
+    //   - "string"
+    //   - union of string literals
```

Comment:
This also seems pretty self-evident based on the types below, so not sure what value this comment is adding.